diesel_cli/src/database_error.rs

```diff
@@ -38,8 +38,8 @@ impl From<result::ConnectionError> for DatabaseError {
 impl Error for DatabaseError {
     fn description(&self) -> &str {
         match *self {
-            CargoTomlNotFound => {
-                "Unable to find Cargo.toml in this directory or any parent directories."
+            ProjectRootNotFound => {
+                "Unable to find diesel.toml or Cargo.toml in this directory or any parent directories."
```

Comment:
It may be useful to indicate what "this" directory is. I don't know much about efficiency if you include a `format!` here, though. That would entail passing in either the url into the error, or something similar, which might cause enum size differences? But I'm not sure.